README.md

```diff
@@ -23,7 +23,8 @@ Vocab helps you ship multiple languages without compromising the reliability of
 Vocab is a monorepo with different packages you can install depending on your usage, the below list will get you started using the cli, React and webpack integrations.
 
 ```bash
-$ npm i --save @vocab/cli @vocab/react @vocab/webpack
+$ npm i --save-dev @vocab/cli
+$ npm i --save @vocab/core @vocab/react
```

Comment:
The line above this should have the reference to `webpack` removed, since that's covered by a later section.